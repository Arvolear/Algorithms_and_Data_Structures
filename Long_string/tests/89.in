5014291
72867044117484298396292816245515011715