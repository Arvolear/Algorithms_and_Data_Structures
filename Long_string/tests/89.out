72867044117484298396292816245520026006
72867044117484298396292816245509997424
365376563514904460089845501864539713607419065
14531873821739563658410095514104